python/kubeflow/trainer/backends/local_process.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2025 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import tempfile
+import venv
+from typing import List, Optional
+
+from kubeflow.trainer.constants import constants
+from kubeflow.trainer.types import types, local as local_types
+from kubeflow.trainer.types.backends import LocalProcessBackendConfig
+from kubeflow.trainer.utils import utils, local as local_utils
+from kubeflow.trainer.backends import base
+from kubeflow.trainer.local import runtimes as local_runtimes
+from kubeflow.trainer.local import job
+
+
+logger = logging.getLogger(__name__)
+
+
+class LocalProcessBackend(base.TrainingBackend):
+
+    def __init__(self,
+                 cfg: LocalProcessBackendConfig,
+                 ):
+        # list of running subprocesses
+        self.__jobs: List[job.LocalJob] = []
+        self.cfg = cfg
+
+
+    def list_runtimes(self) -> List[types.Runtime]:
+        """List of the available Runtimes.
+
+               Returns:
+                   List[Runtime]: List of available local training runtimes.
+                       If no runtimes exist, an empty list is returned.
+               """
+
+        return local_runtimes.runtimes
+
+    def get_runtime(self, name: str) -> Optional[local_types.LocalRuntime]:
```

Comment:
Why get_runtime returns LocalRuntime if interface returns Runtime class: https://github.com/kubeflow/sdk/blob/71e83ae2897cd54aaca23ef5f1585e3577d31d3c/python/kubeflow/trainer/backends/base.py#L29

Reply:
@szaher What additional information might be useful for user while using LocalRuntime compare to Runtime object?